### PR TITLE
[spec] use `KeyForSymbol` from "symbols as weakmap keys" proposal

### PR DESCRIPTION
**Pull request by ljharb**
See https://github.com/tc39/ecma262/pull/2777

### CODE REVIEW

File: spec.emu [hunk 1]
```diff
@@ -27,9 +27,7 @@ contributors:
           1. Let _result_ be Completion(thisSymbolValue(_value_)).
           1. If _result_ is an abrupt completion, return *false*.
           1. Assert: Type(! _result_) is Symbol.
-          1. For each element _e_ of the GlobalSymbolRegistry List (see 20.4.2.2), do
-            a. If SameValue(_e_.[[Symbol]], ! _result_) is true, return *true*.
-          1. Return *false*.
+          1. Return KeyForSymbol(_result_).
```
Comment (acutmore) [inline spec.emu:30]:
```suggestion
          1. Return KeyForSymbol(_result_) is a String.
```

Comment (ljharb) [reply spec.emu:30]: oh right, this will need tweaking :-) good catch

File: index.html [hunk 2]
```diff
@@ -2619,7 +2619,11 @@
 </ul></div><div id="menu-toggle"><svg xmlns="http://www.w3.org/2000/svg" style="width:100%; height:100%; stroke:currentColor" viewBox="0 0 120 120">
       <title>Menu</title>
       <path stroke-width="10" stroke-linecap="round" d="M30,60 h60  M30,30 m0,5 h60  M30,90 m0,-5 h60"></path>
+<<<<<<< HEAD
```
Comment (acutmore) [inline index.html:2622]: mid merge?

Comment (ljharb) [reply index.html:2622]: whoops, lol

**Review by acutmore — APPROVED** [review]

**Pull request merged**